build/DerivedHostMachineInfo.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+    <PropertyGroup>
+      <IsDebianBaseDistro Condition=" '$(HostOSName)' == 'ubuntu' OR '$(HostOSName)' == 'debian' ">true</IsDebianBaseDistro>
+      <IsRPMBasedDistro Condition=" $(HostRid.StartsWith('rhel')) ">true</IsRPMBasedDistro>
```

Comment:
according to this https://github.com/dotnet/corefx/issues/1022 we should use lower case